app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="crayons-layout__sidebar-left">
+  <p class="fw-heavy fs-3xl" style="margin-bottom: 29px;">Mod Center</p>
+  <div id="mod-center-inbox-toggle" style="display: flex; justify-content: space-between; padding: 8px;">
+    <span class="fw-medium">Inbox</span>
+    <span>*</span>
+  </div>
+  <div id="mod-center-inbox" class="hidden">
+    <% if current_user.moderator_for_tags.any? %>
```

Comment:
You can use the instance variable `@tag_moderator_tags` here.

Reply:
👍

Reply:
For some reason Andy, this kept blowing up for me 🤷‍♀️

Reply:
Oops 🤦 I was looking at the wrong controller action...